utilities/pipelines/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -39,6 +39,15 @@ function Invoke-ResourceRemoval {
             }
             break
         }
+        'Microsoft.Authorization/locks' {
```

Comment:
Would you keep this besides the decision out of issue #1493?

Reply:
Not sure I fully crasp the connection. The reason I added the 'case' was the sleep that I did not want to introduce to the general logic (to not waste time).
I even had a while-loop before, but it turned out, the lock is 'considered' removed directly - hence only the sleep seemed to be a viable alternative. Can't say I like it and would have preferred the loop, but it unfortunately didn't do the trick.